src/resources/js/components/Presentation.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div class="container">
+     <div v-if="isError" class="error">
+          <h3>{{ message }}</h3>
+     </div>
+
+     <h3>Presentation info</h3>
+     <div>
+          <h4>title : {{ presentation.title }} </h4>
+          <p>created on : {{ formatDate(presentation.date) }} </p>
+     </div>
+
+  </div>
+</template>
+
+<script>
+import axios from "axios";
+
+export default {
+  name: "Presentation",
+  props: ["idPresentation"], // we will send the id of the presentation through props
+  data() {
+    return {
+      presentation: {},
+      polls: [],
+      isError: false,
+      message: ''
+    };
+  },
+  beforeMount() {
+    this.getPresentationInfo();
+  },
+  methods: {
+     getPresentationInfo() {
+      //const id = idPresentation;
+      const id = 1;
+      const apiUrl = "api/v1/presentations/" + id;
+
+      axios
+        .get(apiUrl)
+        .then(response => {
+          this.presentation = response.data.data;
```

Comment:
Ensure you don't have redundant encapsulation (data.data)

Reply:
right, I rearranged this ambiguous affectation in [42f6bdde](https://github.com/kayoumido/HEIGVD-PRO-A-11/pull/113/commits/42f6bddeafadf1d764b0f3e702d480baa0e4de0e)

Reply:
Thanks